### PR TITLE
add time_feature use/drop option

### PR DESCRIPTION
**Pull request by TheaperDeng**
related to issue #3455

### CODE REVIEW

**Comment by TheaperDeng** [comment]
PR validation (combine) pass:
http://10.239.47.210:18888/job/ZOO-PR-Validation/5068/
http://10.239.47.210:18888/job/ZOO-PR-Python-ExampleTests-py37-ray/1293/

File: pyzoo/zoo/automl/feature/time_sequence.py [hunk 1]
```diff
@@ -36,7 +36,8 @@ def __init__(self, future_seq_len=1,
                  dt_col="datetime",
                  target_col=["value"],
                  extra_features_col=None,
-                 drop_missing=True):
+                 drop_missing=True,
+                 time_attributes=True):
```
Comment (yushan111) [inline pyzoo/zoo/automl/feature/time_sequence.py:40]: time_features?

File: pyzoo/zoo/automl/feature/time_sequence.py [hunk 2]
```diff
@@ -335,8 +337,9 @@ def restore(self, **config):
 
     def get_feature_list(self):
         feature_list = []
-        for feature in (TIME_ATTR + ADDITIONAL_TIME_ATTR):
-            feature_list.append(feature + "({})".format(self.dt_col))
+        if self.time_attributes:
+            for feature in (TIME_ATTR + ADDITIONAL_TIME_ATTR):
```
Comment (yushan111) [inline pyzoo/zoo/automl/feature/time_sequence.py:341]: Where are `TIME_ATTR`, `ADDITIONAL_TIME_ATTR` defined?

**Comment by yushan111** [comment]
LGTM

**Comment by TheaperDeng** [comment]
PR validation (combine) pass:
http://10.239.47.210:18888/job/ZOO-PR-Python-ExampleTests-py37-Spark3/492/
http://10.239.47.210:18888/job/ZOO-PR-Validation/5070/

**Pull request merged**